### ISSUE TITLE
anchorParentSelector item not hidden

### ISSUE THREAD

**Issue by brunob** [goblindegook/littlefoot#17]
Hi, switching from bigfoot to littlefoot, i've adapted the name of few settings i was using as this : 

```javascript
littlefoot.default({
	anchorPattern: /(nb\d+(footnote|appendix))/gi,
	anchorParentSelector: "span",
	footnoteParentClass: "spip_note_ref",
	footnoteSelector: "div"
});
```

Here is the original html of my footnotes anchors : 

```html
<span class="spip_note_ref">&nbsp;[<a href='#nb1' class='spip_note' rel='appendix' title='Outre le Porte-plume déjà intégré dans SPIP 2 et qui initiait ce mouvement, on&nbsp;(...)' id='nh1'>1</a>]</span>
```

And the generated html after littlefoot processing : 

```html
<span class="spip_note_ref">&nbsp;[<span class="littlefoot-footnote__container"><button class="littlefoot-footnote__button littlefoot-footnote__button__ellipsis" id="nh1" data-footnote-button-id="1" data-footnote-number="1" title="See Footnote 1" aria-controls="fncontent:1" aria-expanded="false" aria-label="Footnote 1" rel="footnote"><svg viewBox="0 0 31 6" preserveAspectRatio="xMidYMid"><circle r="3" cx="3" cy="3" fill="white"></circle><circle r="3" cx="15" cy="3" fill="white"></circle><circle r="3" cx="27" cy="3" fill="white"></circle></svg></button></span><a href="#nb1" class="spip_note footnote-print-only" rel="appendix" title="Outre le Porte-plume déjà intégré dans SPIP 2 et qui initiait ce mouvement, on&nbsp;(...)" id="nh1">1</a>]</span>
```

It matkes that the text content of the `<span class="spip_note_ref">` is not hidden, which result to something like `[ button ]` in the page.

The difference i observe is that bigfoot was putting its markup *before* the anchor parent element and hiding it. Instead of this, littlefoot insert its markup *in* the anchor parent element.

Am i missing something or is it a bug in `anchorParentSelector` setting ?

FTR here is a demo page running on bigfoot : https://labo.eliaz.fr/article135.html

**Comment by goblindegook** [comment]
Could be a bug. Thanks for reporting, I will investigate further.

**Comment by goblindegook** [comment]
Fixed for the upcoming 2.0 release.

**Comment by brunob** [comment]
\o/
